src/components/wallet/earn/Validate/AddValidator.vue

```diff
@@ -140,7 +140,7 @@
                                 * {{ $t('staking.validate.summary.warn') }}
                             </label>
                             <p v-if="warnShortDuration" class="err">
-                                {{ $t('staking.validate.errs.duration_warn') }}
+                                <!-- {{ $t('staking.validate.errs.duration_warn') }} -->
```

Comment:
Remove the key from the locale file also

Reply:
ok